### PR TITLE
Beginning of dea access dataloader dag

### PR DESCRIPTION
**Pull request by cyrusbt5**
Added initial dag to setup DEA Access dataloader pipeline in airflow.

Please review @omad @whatnick

### CODE REVIEW

File: dags/ks8_dea_access_dataloader.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+# [END import_module]
+
+# [START default_args]
+# These args will get passed on to each operator
+# You can override them on a per-task basis during operator initialization
+default_args = {
+    'owner': 'Robert Gurtler',
+    'depends_on_past': False,
+    'start_date': days_ago(1),
```
Comment (whatnick) [inline dags/ks8_dea_access_dataloader.py:45]: Use absolute start date

File: dags/ks8_dea_access_dataloader.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          'verify': False,
+        },
+    )
+
+    # [START task_http_egg_svc_check]
+    task_http_egg_svc_check = KubernetesPodOperator(
+        namespace='airflow',
+        name='dea-access-egg-svc-check',
+        task_id='http_egg_svc_sensor_check',
+        hostnetwork=False,
```
Comment (whatnick) [inline dags/ks8_dea_access_dataloader.py:98]: Host network does not need to be specified in k8s.

File: dags/ks8_dea_access_dataloader.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        image_pull_policy='IfNotPresent',
+        image=eggloader_svc_image,
+        is_delete_operator_pod=True,
+        labels={
+          'runner': 'airflow',
+        },
+        env_vars={
+          'DATA_DIR': '{{ params.egg_svc_data_dir }}',
+          'ELASTIC_SCHEME': 'http',
+          'ELASTIC_HOST': '{{ params.egg_svc_name }}',
```
Comment (whatnick) [inline dags/ks8_dea_access_dataloader.py:124]: ES is in a different namespace . Need to ensure connectivity

**Comment by whatnick** [comment]
Please use Upper case for Globals.

**Comment by cyrusbt5** [comment]
> 
> 
> Looks great @cyrusbt5 !
> 
> The only thing I'd add is a reference to where/how/what `EGGLOADER_SVC_IMAGE` is defined and created, since it's a crucial part of this workflow.

Thanks @omad I have added docker image notes. Please also merge as I do not have the power :(

**Pull request merged**